src/evergreen/task.py

```diff
@@ -163,16 +164,6 @@ def log_map(self) -> Dict:
             self._logs_map = {key: value for key, value in self.json["logs"].items()}
         return self._logs_map
 
-    def get_project_identifier(self) -> str:
```

Comment:
Deleting this is a backwards-incompatible change. Which means if anyone is using it and they update to this version, it will break their code. We want to avoid backwards-incompatible changes as much as possible, so user can safely update.

For this, can we keep this function, but just have it return `self.project_identifier`. That way anyone using it should be able to update without any issues.

Reply:
Makes sense. I've added it back in.